README.md

```diff
@@ -1,11 +1,11 @@
 [![Build status][appveyor-badge]][appveyor]
 [![Build Status][travis-ci-badge]][travis-ci]
-[![Join the chat at https://gitter.im/JUCE-cmake/Lobby][gitter-badge]][gitter]
+[![Join the chat at https://gitter.im/FRUT/Lobby][gitter-badge]][gitter]
```

Comment:
frut lobby is not registered

Reply:
I know. I intend to remove it in a subsequent PR, since it's unused.